### DISCUSSION TITLE
The official BeatDrop Music Visualizer website

### DISCUSSION THREAD

**Discussion by OfficialIncubo** [OfficialIncubo/BeatDrop-Music-Visualizer#23] [Future Plans]
Hey! I need a FREE website creator without any watermark and domain watermark in order to make an official Milkdrop Music Visualization Standalone tool website?

Suggestions appreciated! ;)

**Comment by vdmokstati** [comment]
https://vjun.io/vdmo/beatdrop-music-visualiser-with-spout-1hof

Yo.. posted quick announcement for you at vjunion 

For sites .. check this https://render.com/docs/static-sites

**Reply by vdmokstati** [reply]
https://github.com/silexlabs/Silex
Open source website builder .. teleporthq is nice .. 

There are many out there.. figma to code options too

**Comment by OfficialIncubo** [comment]
I am gonna make an official BeatDrop banner when I have time for the VJ Union website.

Plus, I am looking at the Silex later or soon.

**Comment by OfficialIncubo** [comment]
@vdmokstati

I've now launched the official BeatDrop website via GitHub pages in https://officialincubo.github.io/BeatDrop-Music-Visualizer/
Have a look and see if it's good.
